src/ape/managers/chain.py

```diff
@@ -482,7 +482,14 @@ def restore(self, snapshot_id: Optional[SnapshotID] = None):
         self.provider.revert(snapshot_id)
         self.account_history.revert_to_block(self.blocks.height)
 
-    def mine(self, num_blocks: int = 1, timestamp: Optional[int] = None) -> None:
+    def mine(
+        self,
+        num_blocks: int = 1,
+        timestamp: Optional[int] = None,
+        deltatime: Optional[int] = None,
```

Comment:
can we call it `timedelta`? Does that still read well?
```suggestion
        timedelta: Optional[int] = None,
```

Reply:
I didn't use timedelta because it is a class within datetime, and I don't want any confusion

Reply:
I think we should leave it as deltatime, but you make the last decision on that